ext/zend_test/php_test.h

```diff
@@ -59,6 +60,8 @@ ZEND_BEGIN_MODULE_GLOBALS(zend_test)
 	zend_long quantity_value;
 	zend_string *str_test;
 	zend_string *not_empty_str_test;
+	int zendmm_observer_enabled;
+	zend_mm_observer *observer;
```

Comment:
Nit: for consistency, could you rename `zendmm_` symbols and files to `zend_mm_` ?